app/components/search/ResultsBox.vue

```diff
@@ -98,14 +98,12 @@
           </td>
         </tr>
       </tbody>
-
-      <div
-        class="absolute bottom-1/2 right-1/2 translate-x-1/2 translate-y-1/2 transform"
-      >
-        <LoadingSpinner v-if="search.isLoading" />
-      </div>
     </table>
 
+    <div class="absolute bottom-1/2 right-1/2 translate-x-1/2 translate-y-1/2 transform">
+      <LoadingSpinner v-if="search.isLoading" />
+    </div>
+
     <PopupDialog :show="showDateDialog">
```

Comment:
fixed the warning: <div> cannot be child of <table>, according to HTML specifications. This can cause hydration errors or potentially disrupt future functionality.